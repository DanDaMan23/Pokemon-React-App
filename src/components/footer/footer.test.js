import { fireEvent, render, screen } from "@testing-library/react"
import Footer from "./footer"

describe("Footer Component", () => {
  const component = <Footer />

  it("renders the component", () => {
    // Arrange
    render(component)

    // Act

    // Assert
    expect(component).toMatchSnapshot()
  })

  it("scrolls up when scroll top is clicked", () => {
    // Arrange
    render(component)

    // Act
    const buttonElement = screen.getByTestId("scroll-top-button")

    // Assert
    fireEvent.scroll(buttonElement, { target: { scrollX: 0, scrollY: 0 } })
  })
})
