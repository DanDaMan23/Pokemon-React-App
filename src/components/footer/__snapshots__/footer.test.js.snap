// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders the component 1`] = `<Footer />`;
